Doc/library/stdtypes.rst

```diff
@@ -5496,6 +5496,15 @@ types, where they are relevant.  Some of these are not reported by the
    .. versionadded:: 3.3
 
 
+.. attribute:: class.__fully_qualified_name__
```

Comment:
Can you link this from the glossary entry for "qualified name"? Maybe even add a separate glossary entry for "fully qualified name".

Reply:
Sure, I added "fully qualified name" term to the documentation glossary. You can see that the term is used for many different things, especially "module fully qualified name" in import/importlib documentation.